### PR TITLE
I had added README.md please review and merge

### PR DESCRIPTION
**Pull request by ivedantmistry**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+Note: Make sure you have the Qt5 library installed on your system before running the calculator. You can install it using the following command:
+```
+pip install pygame
+```
+
+3. Run the calculator:
+- If you have Python installed, run the following command:
+  ```
+  python unserviceable.py
```
Comment (farisfaikar) [inline README.md]: `python Unserviceable.py`. It uses capital U at the beginning

File: README.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+  ```
+
+Note: Make sure you have the pygame library available in your Python environment.
+
+## Future Enhancements
+
+In the future, I plan to enhance this logarithm calculator by adding more advanced scientific calculator functionalities, including:
```
Comment (farisfaikar) [inline README.md]: It's not a logarithm calculator XD

**Review by farisfaikar — CHANGES_REQUESTED** [review]
There are some errors, it'd be great if you correct them.

**Review by farisfaikar — APPROVED** [review]
Cool! Thanks. Will merge now

**Comment by ivedantmistry** [comment]
you did it right now?

**Pull request merged**